data cleaning labeling.ipynb

```diff
@@ -0,0 +1,1784 @@
+{
```

Comment:
<p>Move this cell up to where you label the values or maybe even to the readme</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>